src/modality-agnostic-files.md

```diff
@@ -169,7 +169,7 @@ and does not validate the syntax of Markdown and reStructuredText.
 The `README` file SHOULD be structured such that its contents can be easily understood
 even if the used format is not rendered.
 A guideline for creating a good `README` file can be found in the
-[bids-starter-kit](https://github.com/bids-standard/bids-starter-kit/blob/master/templates/README).
+[bids-starter-kit](https://github.com/bids-standard/bids-starter-kit/blob/main/templates/README.MD).
```

Comment:
What do you think about pointing people to the templates directory, where the README will be displayed anyway?

```suggestion
[bids-starter-kit](https://github.com/bids-standard/bids-starter-kit/tree/main/templates/).
```

Reply:
Great idea!